packages/lmnt/src/lmnt-speech-model.ts

```diff
@@ -0,0 +1,180 @@
+import { SpeechModelV1, SpeechModelV1CallWarning } from '@ai-sdk/provider';
+import {
+  combineHeaders,
+  createBinaryResponseHandler,
+  parseProviderOptions,
+  postJsonToApi,
+} from '@ai-sdk/provider-utils';
+import { z } from 'zod';
+import { LMNTConfig } from './lmnt-config';
+import { lmntFailedResponseHandler } from './lmnt-error';
+import { LMNTSpeechModelId } from './lmnt-speech-settings';
+import { LMNTSpeechAPITypes } from './lmnt-api-types';
+
+// https://docs.lmnt.com/api-reference/speech/synthesize-speech-bytes
+const lmntSpeechCallOptionsSchema = z.object({
+  model: z.enum(['aurora', 'blizzard']).optional().default('aurora'),
+  language: z
+    .enum([
+      'auto',
+      'en',
+      'es',
+      'pt',
+      'fr',
+      'de',
+      'zh',
+      'ko',
+      'hi',
+      'ja',
+      'ru',
+      'it',
+      'tr',
+    ])
+    .optional()
+    .default('auto'),
+  format: z
+    .enum(['aac', 'mp3', 'mulaw', 'raw', 'wav'])
+    .optional()
+    .default('mp3'),
+  sampleRate: z.number().int().optional().default(24000),
+  speed: z.number().min(0.25).max(2).optional().default(1),
+  seed: z.number().int().optional(),
+  conversational: z.boolean().optional().default(false),
+  length: z.number().max(300).optional(),
+  topP: z.number().min(0).max(1).optional().default(1),
+  temperature: z.number().min(0).optional().default(1),
+});
```

Comment:
consider comments & also the enum + string trick Sam mentioned for better autocomplete DX